src/material/checkbox/_checkbox-theme.scss

```diff
@@ -12,9 +12,11 @@
   $warn: map.get($config, warn);
   $foreground: map.get($config, foreground);
 
-  .mat-mdc-checkbox {
+  @at-root #{& or html} {
```

Comment:
I think the convention is to emit under `:root` rather than `html`. They should be targeting the same element, but I think the specificity may be different.

Reply:
`:root` is more specific, and I'd actually like to target the least specific thing possible so that users don't have trouble overriding. There's also an interesting google3-specific issue because CSS vars targeted at `:root` are treated as serve-time variables by sass stack [go/sass-stack-variables](https://goto.google.com/sass-stack-variables)